src/ray/rpc/grpc_server.cc

```diff
@@ -42,6 +44,8 @@ GrpcServer::GrpcServer(std::string name,
   // Enable built in health check implemented by gRPC:
   //   https://github.com/grpc/grpc/blob/master/doc/health-checking.md
   grpc::EnableDefaultHealthCheckService(true);
+  grpc::reflection::InitProtoReflectionServerBuilderPlugin();
```

Comment:
Can we make this optional? I will be surprised if this has no perf impact. Maybe we can allow to enable it using env var?

Reply:
or do a quick micro benchmark?  actually i think it's unlikely to affect normal RPCs.